CMakeLists.txt

```diff
@@ -159,7 +182,6 @@ if(ENABLE_LZ4_EXT)
 endif()
 # }
 
-option(RDKAFKA_BUILD_STATIC "Build static rdkafka library" OFF)
```

Comment:
Is it absolutely required to build both static and dynamic for CPack? The reason I ask is that removing this option will conflict with changes in https://github.com/edenhill/librdkafka/pull/3130, particularly https://github.com/edenhill/librdkafka/pull/3130/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20a

Reply:
I guess not. CPacking will work as long as the static targets are named differently from the dynamic ones.